File: JavaScript/8js 3/validateCode.js
/*
|--------------------------------------------------------------------------
| validate code with simple regex
|--------------------------------------------------------------------------
|
*/

function validateCode(code) {
   return "123".includes(String(code)[0]);
}


function validateCode(code) {
   return /^[1-3]/.test(code);
}


function validateCode(code) {
   return +(code + "")[0] < 4;
}


console.log(validateCode(123), true);
console.log(validateCode(248), true);
console.log(validateCode(8), false);
console.log(validateCode(321), true);
console.log(validateCode(9453), false);

File: Python/8py 3/validate_code.py
"""
|--------------------------------------------------------------------------
| validate code with simple regex
|--------------------------------------------------------------------------
|
"""

def validate_code(code):
   return str(code)[0] in '123'


from re import match
def validate_code(code):
   return bool(match('[123]',str(code)))


def validate_code(code):
   return int(str(code)[0]) < 4


print(validate_code(123), True)
print(validate_code(248), True)
print(validate_code(8), False)
print(validate_code(321), True)
print(validate_code(9453), False)